web/src/stories/index.js

```diff
@@ -32,3 +32,30 @@ storiesOf('Buttons', module).add('various', () => (
     <Btn kind="outline" /> <Btn kind="outline" size="big" />
   </div>
 ));
+
+storiesOf('Numeric inputs', module).add('Dollars', () => {
+  class Dollar extends Component {
+    state = { value: 0 };
+
+    change = e => {
+      console.log(e.target.value);
+      this.setState({ value: e.target.value });
+    };
+
+    render() {
+      return (
+        <DollarInput
+          hideLabel
+          wrapperClass="m0"
+          className="fake-spacer-input m0 input input-condensed mono right-align"
```

Comment:
Oh, shoot, I forgot to remove the `fake-spacer-input` from the classlist when I merged the table PR. There isn't an associated class, so you can drop it from here.